heron/tools/cli/src/python/response.py

```diff
@@ -52,10 +54,10 @@ class Response(object):
      If the response object is generated by a statement that
      shells out program, `msg` is stdout, `detailed_msg` is stderr
   """
-  def __init__(self, status_code, msg=None, detailed_msg=None):
-    self.status = status_type(status_code)
-    self.msg = msg
-    self.detailed_msg = detailed_msg
+  def __init__(self, proc=None, status=None):
+    # invariant: only one of proc and status is None
```

Comment:
Is there some python magic enforcing this? If not, let's explicitly check the state.